draft-ietf-sedate-datetime-extended.md

```diff
@@ -401,28 +397,24 @@ mark (see `critical-flag` in {{abnf}}).
 
 IXDTF strings such as:
 
-    2022-07-08T00:14:07Z[Europe/Paris]
     2022-07-08T00:14:07+01:00[Europe/Paris]
 
 are internally inconsistent, as Europe/Paris does not use a time zone
-offset of 0 (which is indicated in the `Z`, an abbreviation for
-`+00:00`), nor a time zone offset of `+01:00` in July 2022.
+offset of +01:00 in July 2022.
 The time zone hint given in the suffix tag is elective, though, so the recipient is not
 required to act on the inconsistency; it can treat the Internet
 Date/Time Format string as if it were:
 
-    2022-07-08T00:14:07Z
     2022-07-08T00:14:07+01:00
 
 Similar with:
 
-    2022-07-08T00:14:07Z[knort=blargel]
+    2022-07-08T00:14:07+01:00[knort=blargel]
```

Comment:
I had a slightly different approach in my PR.  Please compare.

Reply:
Cool, your PR's version looks good to me. Apologies for the overlap, I foolishly didn't check new PRs because I didn't know you'd work so fast. Nice work!

Do you want to merge any changes from this PR into yours that you think would be appropriate?  Or should we keep them separate? I'm flexible either way.

Reply:
BTW, I just pushed a new commit that removed the divergence (I think).